app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -58,6 +60,26 @@ def get_vms
     process_collection(@inv[:vms], :vms) { |vm| parse_vm(vm) }
   end
 
+  def get_private_images
+    @inv[:orgs].each do |org|
+      org.catalogs.each do |catalog|
+        get_images(catalog.catalog_items) unless catalog.is_published
```

Comment:
Better to build up `@inv[:images]` or `@inv[:catalogs]` here and call `get_images` once, I've seen some assumptions in the `get_xyz` methods that they have all of the inventory already.  Might not be a problem now but could bite someone down the road.